Gemfile

```diff
@@ -52,9 +52,9 @@ group :development, :test do
 end
 
 group :test do
-  gem 'rspec-rails', require: false
-  gem 'factory_girl', require: false
-  gem 'capybara', require: false
+  gem 'rspec-rails'
+  gem 'factory_girl_rails'
+  gem 'capybara'
```

Comment:
possibly because if someone is doing a bundle install mistakenly with the test group when they don't need it, it's okay?   The whole "require: false" thing is documentation I've encountered elsewhere for rspec or coveralls or something like that and I suspect it's just a pattern the coder was used to.